modules/ROOT/pages/microprofile-config-properties.adoc

```diff
@@ -282,6 +288,109 @@ This property is available starting in MicroProfile Metrics 5.0 .
 |Server startup
 |`mp.metrics.tags=app=shop,tier=integration,special=deli\=ver\,y`
 
+|`mp.metrics.distribution.percentiles`
+a|Specifies a custom set of percentiles for matching Histogram and Timer metrics to track and output. Can be used to disable percentile output.
```

Comment:
The Histogram and Timer metric output data related to percentiles (of the recorded data).
There is a default set of percentiles (50th, 75th, 95th, 98th, 99th and 99.9th percentiles)
The property allows users to define their own custom set of percentiles that are tracked and then output.

Regarding the "Matching".
The properties follow a <property_name>=<metric_name1>=<values1>';<metric_name2>=<values2>';...;<metric_nameN>=<valuesN>'

The "matching" metric is matching the metric name defined in the property.
The bullet points were there to hopefully define what that meant (plus the example)

Reply:
Ok- so the metrics that "match" with the metric name defined in the property. I think this could be easy to misread as saying the timer metrics match the histogram metrics. AT least that's what I first thought.

Would this then also be correct?

> Defines a custom set of percentiles for the specified histogram and timer metrics to track and output.

Reply:
Yep, that sounds more clear than what I had 😂

Reply:
Also, I guess we can probably remove the last sentence in favor of the last bullet point below:

`Setting the property without any value disables percentiles for all histogram and timer metrics.`